web/src/AnalyticsNudge.tsx

```diff
@@ -2,6 +2,19 @@ import React, { Component } from "react"
 import "./AnalyticsNudge.scss"
 
 class AnalyticsNudge extends Component {
+  static analyticsOpt(optIn: boolean) {
+    let url = `//${window.location.host}/api/analytics/opt`
```

Comment:
Should this use the `PathBuilder`?

Reply:
i mean, i don't think so, since it is talking directly to whatever is hosting the page, no need to do weird relativism stuff here -- but i think more importantly, the path builder was relevant for the Sail server since we had to make paths relevant to something other than we had been, but this nudge should never show up on a page served by sail. i'll file a ticket.